src/tracker/mod.rs

```diff
@@ -412,7 +413,7 @@
 //!
 //! For example, the HTTP tracker would send an event like the following when it handles an `announce` request received from a peer using IP version 4.
 //!
-//! ```rust,ignore
+//! ```text
```

Comment:
Why the changing from `rust,ignore` to `text`?

Reply:
I do not want to compile that piece of code, and I do not want to have an "ignore" test in the output like this:

```
& cargo test --doc
test src/tracker/mod.rs - tracker (line 83) - compile ... ignore
```

Sometimes the necessary setup to compile is too verbose for the documentation. For example:

```text
let announce_data = tracker.announce(&info_hash, &mut peer, &peer_ip).await;
```

That line would require building a tracker, a peer, etc.

We have the tests if you want to check a fully operational example; maybe I could add a link to the tests in these cases. 

I can also implement the full working example and [hide the lines that are not interesting for the documentation](https://rust-lang.github.io/mdBook/format/mdbook.html#hiding-code-lines).

Reply:
The link to how to hide code in docs is wrong. I've read somewhere else that you can do that, but I do not find it.

Reply:
The link was not wrong it's only that you need to prepend the hashtag on all lines:

```text
//! ```rust,no_run
//! # use torrust_tracker::tracker::peer;
//! # use torrust_tracker::shared::bit_torrent::info_hash::InfoHash;
//! # // ...
//! 
//! let announce_data = tracker.announce(&info_hash, &mut peer, &peer_ip).await;
//! ```
```

That would show only:

```rust

let announce_data = tracker.announce(&info_hash, &mut peer, &peer_ip).await;
```